apps/aecore/src/aec_db.erl

```diff
@@ -558,13 +564,18 @@ gc_tx(TxHash) ->
            BlockHash when is_binary(BlockHash) ->
                {error, BlockHash};
            mempool ->
-               delete(aec_tx_pool, TxHash);
+               delete(aec_tx_pool, TxHash),
+               delete(aec_signed_tx, TxHash),
+               write(aec_tx_gc, #aec_tx_gc{key = TxHash, value = 0});
```

Comment:
Is this 0 a placeholder? May I suggest using a macro for readability?

See also https://github.com/aeternity/aeternity/pull/2329/files

Reply:
Will do. I will also use `[]` as a placeholder, as it is not for `aec_tx_pool` objects.